go/chat/utils/utils.go

```diff
@@ -3052,3 +3053,26 @@ func IsDeletedConvError(err error) bool {
 		return false
 	}
 }
+
+func TextPaymentsToTransactionIDs(textPayments []chat1.TextPayment) (txIDs []stellar1.TransactionID) {
+	for _, payment := range textPayments {
+		if payment.Result.IsTyp(chat1.TextPaymentResultTyp_SENT) {
+			txID := stellar1.TransactionIDFromPaymentID(payment.Result.Sent())
+			txIDs = append(txIDs, txID)
+		}
+	}
+	return txIDs
+}
+
+func GetMessageBodyTxIDs(mtype chat1.MessageType, body chat1.MessageBody) []stellar1.TransactionID {
```

Comment:
currently not handling edit message types, since new stellar payments are not made on edit messages.